### ISSUE TITLE
The engine is searching for the `tzdb` at a wrong location

### ISSUE THREAD

**Issue by nico-stefani** [wazuh/wazuh#26950]
## Description

During the development of #26908, we detected that at the start `wazuh-engine` is trying to read the `tzdb` from `/usr/bin` instead of the default  `/var/lib/wazuh-server/engine/tzdb`.

```console
[root@3393f4a18422 pkg]# su wazuh -c 'wazuh-engine server -l info start'
2024-11-21 15:17:32.690 24562:24562 info: Logging initialized.
2024-11-21 15:17:32.693 24562:24562 info: Store initialized.
2024-11-21 15:17:32.693 24562:24562 info: RBAC initialized.
2024-11-21 15:17:32.693 24562:24562 info: MetricsManager: Created new scope: (KVDB)
2024-11-21 15:17:32.774 24562:24562 info: KVDB initialized.
2024-11-21 15:17:32.774 24562:24562 info: Geo initialized.
2024-11-21 15:17:32.788 24562:24562 info: Schema initialized.
2024-11-21 15:17:32.788 24562:24562 warning: Failed to load timezone database: 'Timezone database not found at "/usr/bin/tzdb"', try to download it
mkdir: cannot create directory '/usr/bin/tzdb': Permission denied
2024-11-21 15:17:32.792 24562:24562 warning: Failed to download timezone database: ''
2024-11-21 15:17:32.792 24562:24562 error: An error occurred while initializing the modules: Timezone database not found at "/usr/bin/tzdb".
2024-11-21 15:17:32.794 24562:24562 info: KVDB terminated.
```

Is needed to use the default to run the whole Wazuh server without the necessity of the `root` user.

**Comment by JcabreraC** [comment]
It seems that an outdated package was used during the test.

Previously, it was relative to the binary. However, it now defaults to a different value, as seen here:
https://github.com/wazuh/wazuh/blob/6a5f715bdf763d40d16a5be3d234369bc060d56e/src/engine/source/cmds/src/defaultSettings.hpp#L47

**Comment by JcabreraC** [comment]
This has been resolved in PR [#26355](https://github.com/wazuh/wazuh/pull/26355), which has already been merged.
